interfaces/AW-App/src/app/services/logging.service.ts

```diff
@@ -18,7 +18,7 @@ export class LoggingService {
         connectionString: `InstrumentationKey=${environment.ai_ikey};IngestionEndpoint=${environment.ai_endpoint}`,
         instrumentationKey: environment.ai_ikey,
         enableAutoRouteTracking: true,
-        disableCookiesUsage: true,
+        // disableCookiesUsage: true,
```

Comment:
Why is this?

I think you don't have the latest version of AI-sdk installed locally...